Samples/Mica/cpp-uwp-WinUI2/UWP WinUI2/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.UI.Xaml" version="2.8.0-prerelease.210927001" targetFramework="native" />
-  <package id="Microsoft.Web.WebView2" version="1.0.1018-prerelease" targetFramework="native" />
+  <package id="Microsoft.Web.WebView2" version="1.0.1083-prerelease" targetFramework="native" />
 </packages>
```

Comment:
prerelease?

Reply:
Would this repo benefit from the DevCheck -CheckDependencies / MasterSourceOfTruth/master-dependency-definition/verification like we added recently in WinAppSDK?

Will copied `DevCheck.*` to `eng\*` to propogate everywhere via Maestro, of urgent need to leverage some functionality and more longer-term. That's currently an older copy as there were changes pending. I've got a change going in RSN to finish that (moving Foundation's (latest) copy to `eng\`) so it should be available for use in this repo if we want.

Do we want?